src/makeRobotAccountant.js

```diff
@@ -7,6 +7,20 @@
 
 function makeRobotAccountant() {
   // write code here
+  let i = 1;
+
+  return function(a) {
+    return function(b) {
+      if (i > 3 && i % 2 === 0) {
```

Comment:
Here,` i % 2 == 0` may be unnecessary because our function will produce incorrect results three times.

Reply:
that's not true, task states that we need to allow 3 times and then every odd call returns value and even returns bzz